File: search/inode_search.py
import os
from search.single_repo import enum_filename


inodes= {
    'inode':[]
}


def search_inodes(repo_dir_path:str) -> dict:
    
    vscode_dir = enum_filename(os.path.join(repo_dir_path),'.vscode',False)
    if len(vscode_dir) > 0:
            print('Found vscode dir')
            

File: src/search/inode_search.ts
import { join } from "path";
import { enumFilename } from "./single_repo";

const inodes = {
    inode: []
};

function search_inodes(repo_dir_path: string) {
    const vscode_dir = enumFilename(join(repo_dir_path), '.vscode', false);
    if (vscode_dir.length > 0) {
        console.log('Found vscode dir');
    }
}